bitnami/airflow/values.yaml

```diff
@@ -1,67 +1,241 @@
+## @section Global parameters
 ## Global Docker image parameters
 ## Please, note that this will override the image parameters, including dependencies, configured to use the global value
-## Current available global Docker image parameters: imageRegistry and imagePullSecrets
+## Current available global Docker image parameters: imageRegistry, imagePullSecrets and storageClass
+
+## @param global.imageRegistry Global Docker image registry
+## @param global.imagePullSecrets Global Docker registry secret names as an array
+## @param global.storageClass Global StorageClass for Persistent Volume(s)
 ##
-# global:
-#   imageRegistry: myRegistryName
-#   imagePullSecrets:
-#     - myRegistryKeySecretName
-#   storageClass: myStorageClass
+global:
+  imageRegistry:
+  ## e.g.
+  ## imagePullSecrets:
+  ##   - myRegistryKeySecretName
+  ##
+  imagePullSecrets: []
+  storageClass:
+
+## @section Common parameters
 
-## Force target Kubernetes version (using Helm capabilites if not set)
+## @param affinity Affinity for pod assignment (evaluated as a template)
+## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+## Note: podAffinityPreset, podAntiAffinityPreset, and  nodeAffinityPreset will be ignored when it's set
+##
+affinity: {}
```

Comment:
Would make sense now that we will modify every `values.yaml` to alphabetically order the parameters inside the sections? It could make diffing two versions easier but also would break logical grouping of parameters.

Reply:
Could you elaborate on the advantages of sorting the parameters?

From what I have seen on this PR, it is more difficult to review the metadata changes since almost the full `values.yaml` is changing. Do you think it will have advantages in the future?

Reply:
After discussion, we are going to keep the original ordering of the `values.yaml` files. The work to order them doesn't compensate the benefits and makes reviewing this PRs a pain.